src/Components/Blazor/Build/test/RuntimeDependenciesResolverTest.cs

```diff
@@ -128,13 +121,21 @@ public void FindsReferenceAssemblyGraph_ForStandaloneApp()
 
             var expected = new HashSet<string>(expectedContents);
             var actual = new HashSet<string>(contents);
-            _output.WriteLine("Expected contents to have:");
-            _output.WriteLine(string.Join(",", expected.Except(actual)));
 
-            _output.WriteLine("Unexpected contents:");
-            _output.WriteLine(string.Join(",", actual.Except(expected)));
+            var contentNotFound = expected.Except(actual);
+            var additionalContentFound = actual.Except(expected);
 
             // Assert
+            if (contentNotFound.Any())
+            {
+                Assert.False(true, $"Expected content not found: {string.Join(", ", contentNotFound)}");
+            }
+
+            if (additionalContentFound.Any())
+            {
+                Assert.False(true, $"Unexpected content found: {string.Join(", ", additionalContentFound)}");
+            }
```

Comment:
this is less-good now right? It will only log one or the other. 

I always end up writing exceptions that derive from XunitException for things like this.